charts/aws-mountpoint-s3-csi-driver/values.yaml

```diff
@@ -103,5 +103,8 @@ awsAccessSecret:
   accessKey: access_key
   sessionToken: session_token
 
+eksPodIdentityAgent:
+  ipv4TargetHost: "169.254.170.23"
```

Comment:
Should we just accept full URI here to not deal with formatting later?

Reply:
Done, thanks!